src/plugin/models/generalModels.ts

```diff
@@ -36,3 +36,13 @@ export enum AlgorandUnit {
 /** Algorand general value type */
 export type AlgorandValue = string | number | Buffer | Uint8Array
 export type AlgorandSymbol = string & Models.ChainSymbolBrand
+
+/** SignMethod enum contains the methods that can be used to sign a message */
+export enum SignMethod {
```

Comment:
since SignMethod and AlgoSignDataInput are both only used for signMessage, this should be clear here. please rename:
SignMethod => AlgorandSignMessageMethod
AlgoSignDataInput => AlgorandSignMessageDataInput 
note: please dont shorten the prefix from Algorand - as 'Algo' is the currency name and can add confusion

Reply:
Done!
Noted, I got confused with the file naming convention, will keep this in mind.